src/Raven.Server/Web/System/CompareExchangeHandler.cs

```diff
@@ -34,6 +34,32 @@ public Task GetCompareExchangeValues()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/databases/*/cmpxchg/indexes", "GET", AuthorizationStatus.ValidUser)]
+        public Task GetCompareExchangeIndexes()
```

Comment:
what is this for?

Reply:
If I want to update several compare exchange values.. So I need to know only their indexes.